web/test/pages/roadmap.test.tsx

```diff
@@ -228,4 +239,132 @@ describe("roadmap page", () => {
     expect(within(sectionStart).getByText("step2")).toBeInTheDocument();
     expect(within(sectionStart).getByText("step4")).toBeInTheDocument();
   });
+
+  it("displays sections based on default state", () => {
+    useMockRoadmap({
+      steps: [
+        generateStep({
+          name: "step1",
+          section: "PLAN",
+          tasks: [generateTask({ id: "task1" })],
+        }),
+        generateStep({
+          name: "step2",
+          section: "START",
+          tasks: [generateTask({ id: "task2" })],
+        }),
+      ],
+    });
+
+    useMockUserData({
+      preferences: generatePreferences({
+        roadmapOpenSections: ["PLAN", "START"],
+      }),
+    });
+
+    const subject = renderRoadmapPage();
+
+    const sectionStart = subject.getByTestId("section-start");
+    const sectionPlan = subject.getByTestId("section-plan");
+
+    expect(within(sectionStart).getByText("step2")).toBeVisible();
+    expect(within(sectionPlan).getByText("step1")).toBeVisible();
+  });
+
+  it("expands start section only", () => {
+    useMockRoadmap({
+      steps: [
+        generateStep({
+          name: "step1",
+          section: "PLAN",
+          tasks: [generateTask({ id: "task1" })],
+        }),
+        generateStep({
+          name: "step2",
+          section: "START",
+          tasks: [generateTask({ id: "task2" })],
+        }),
```

Comment:
duplicating this task setup over multiple tests is a great opportunity to put them all in a describe block with a `beforeEach` to do shared setup

Reply:
Addressed.